### ISSUE TITLE
block number in 1.3.

### ISSUE THREAD

**Issue by Edwardmark** [PKU-YuanGroup/Open-Sora-Plan#495]
Hi, thanks for your great work! What is the number of blocks in v1.3? is it still the same of v1.2? It seems strage not double the block number like stdit.

**Comment by LinB203** [comment]
Yes, since we adopted the 3D attention architecture for 1.2, we don't need to double the number of layers.
You can find the number of layers [here](https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/main/opensora/models/diffusion/opensora_v1_3/modeling_opensora.py#L294).
